lib/random_selection.rb

```diff
@@ -0,0 +1,8 @@
+class RandomSelection
+  attr_reader :rps
+
+  def rps
+    @rps = ["Rock", "Paper", "Scissors"].sample
```

Comment:
I'd suggest moving this list of strings to a class constant such as `POSSIBLE_SELECTIONS` to make it clear it's some "configuration" that won't be changed when the program is running